hermes-console/static/js/console/subscription/SubscriptionController.js

```diff
@@ -252,7 +252,7 @@ subscriptions.controller('SubscriptionController', ['SubscriptionRepository', 'S
         };
 
         $scope.debugFilters = function () {
-            filtersDebuggerModal.open(topicName, $scope.subscription.filters)
+            filtersDebuggerModal.open(topicName, $scope.subscription.filters, topicContentType)
```

Comment:
Does it work? There is no local variable named `topicContentType` in scope of this controller. In my opinion you should use `$scope.topicContentType`.

Reply:
1. The #1304 issue has not been described in detail and it was obvious that validating json message did not work. It was natural to assume that the issue was to provide a support for debugging json messages (fixing bug). I will create another issue regarding the formatting so that it can be tackled in the future.
2. It seems to work. I have added the prefix because maybe I have not tested all possible scenarios.